pkg/test/useraccount/doubles.go

```diff
@@ -0,0 +1,40 @@
+package uatest
+
+import (
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-common/pkg/condition"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type UaModifier func(ua *toolchainv1alpha1.UserAccount)
+
+func NewUserAccountFromMur(mur *toolchainv1alpha1.MasterUserRecord, modifiers ...UaModifier) *toolchainv1alpha1.UserAccount {
+	ua := &toolchainv1alpha1.UserAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      mur.Name,
+			Namespace: test.MemberOperatorNs,
+		},
+		Spec: mur.Spec.UserAccounts[0].Spec,
+	}
+	ModifyUa(ua, modifiers...)
+	return ua
+}
+
+func ModifyUa(ua *toolchainv1alpha1.UserAccount, modifiers ...UaModifier) {
```

Comment:
should we make this function private?

Reply:
Why? It's used in the test: https://github.com/codeready-toolchain/host-operator/blob/master/pkg/controller/masteruserrecord/sync_test.go#L85

Reply:
ah ok, I thought it was only used in the `NewUserAccountFromMur` func. 
but since it's in the `useraccounts` pkg, could we just rename it to `Modify`?

Reply:
done 63785f9